_layouts/default.html

```diff
@@ -11,10 +11,33 @@
           integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ="
           crossorigin="anonymous" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-
+    <link rel="stylesheet" href="/assets/css/styles.css">
+    <script src="/assets/js/responsive-nav.js"></script>
```

Comment:
This now references a nonexisting location.

Reply:
ups ^^

Reply:
So, I hope I fixed it.